dune/stuff/common/configuration.hh

```diff
@@ -82,16 +91,15 @@ static const bool configuration_log_on_exit = false;
 static const std::string configuration_logfile = "data/log/dsc_parameter.log";
 
 
-template <class T>
+template < class T >
 struct Typer {
-  typedef typename std::conditional<std::is_convertible<T, std::string>::value, std::string, T>::type type;
+  typedef typename std::conditional< std::is_convertible< T, std::string >::value, std::string, T >::type type;
 };
 
 } // namespace internal
 
 
-class Configuration
-  : public Dune::ParameterTree
+class Configuration : public Dune::ParameterTree
```

Comment:
This is a complete no-go for me. I could not find the appropriate option, someone else perhaps?